modules/signing_worker/manifests/system_user.pp

```diff
@@ -1,18 +1,18 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-class roles_profiles::profiles::cltbld_user {
-
+class signing_worker::system_user (
```

Comment:
Uh, lots of other roles_profiles use `include ::roles_profiles::profiles::cltbld_user`, so won't this file move break them ?

Reply:
hmm, it didn't flag up any errors when running it. Odd. It seems it's being used for multiple purposes now, such as running a generic worker. What's the intended purpose of this user? Perhaps we should rename the one that's not for scriptworkers.

Reply:
There might only be errors when those other roles attempt to include cltbld_user. Worst case you can restore the original file, right ?